<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExteriorWidget</class>
 <widget class="QWidget" name="ExteriorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="group_text">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>571</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>文本属性</string>
   </property>
   <widget class="QComboBox" name="combo_Language">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>21</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_type_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>23</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>语言：</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="edit_text">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>75</y>
      <width>341</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="plainText">
     <string>输入文本</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_Alignment">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>居中</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>左对齐</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>右对齐</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_type_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>84</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>对齐：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_type_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>144</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>文本色：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_color">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_type_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>53</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>字体：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_Font">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_type_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>112</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>大小：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_Size">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>128</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_bold">
    <property name="geometry">
     <rect>
      <x>221</x>
      <y>48</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>B</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Italic">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>48</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>I</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_underline">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>48</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>U</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_copyText">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>17</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>从0状态复制</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_copyText_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>17</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>从第一种语言复制</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="group_Pic">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>571</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>外形属性</string>
   </property>
   <widget class="QGraphicsView" name="View">
    <property name="geometry">
     <rect>
      <x>436</x>
      <y>20</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_no">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>86</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>不使用图片</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_copyPic">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>401</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>从0状态复制</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_lib">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>56</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>选择图库</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_picture">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>116</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>选择外部图片</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_color_2">
    <property name="geometry">
     <rect>
      <x>292</x>
      <y>30</y>
      <width>138</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_path">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>118</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_backcolor">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>58</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_color">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>58</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>背景色：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Trans">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>88</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>透明度：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_Trans">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>88</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
   </widget>
  </widget>
  <widget class="QSpinBox" name="spin_StatusIndex">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_type_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>13</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>选择状态</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_Flick">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>闪烁</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_flick">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>文本闪烁</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>状态切换</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>控件显隐</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
